crates/server-primitives/src/admin.rs

```diff
@@ -210,13 +210,19 @@ pub struct NearSignatureMessageMetadata {
 #[derive(Clone, Copy, Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
 #[non_exhaustive]
-#[allow(clippy::empty_structs_with_brackets)]
+#[expect(
+    clippy::empty_structs_with_brackets,
+    reason = "Needed for serialisation"
+)]
```

Comment:
ideally, lints shouldn't distract from functionality, move the functional macro annotations closer to the struct